feature-libs/checkout/base/components/checkout-payment-method/checkout-payment-method.component.html

```diff
@@ -28,7 +29,10 @@ <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
           class="cx-payment-card col-md-12 col-lg-6"
           *ngFor="let card of cards; let i = index"
         >
-          <div class="cx-payment-card-inner">
+          <div
+            class="cx-payment-card-inner"
+            (click)="selectPaymentMethod(card.paymentMethod)"
```

Comment:
Nice. Previously we had to click on the actual text, right? And now it's possible to click on the whole card?

Reply:
yup